From 287b10714e654ee1d06d254f89bc9b63a6519121 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob-chen@iotwrt.com>
Date: Thu, 9 Nov 2017 10:32:00 +0800
Subject: [PATCH] enable 1500000 to baud speed table

Signed-off-by: Jacob Chen <jacob-chen@iotwrt.com>
---
 libbb/speed_table.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/libbb/speed_table.c b/libbb/speed_table.c
index 11ced01..079add6 100644
--- a/libbb/speed_table.c
+++ b/libbb/speed_table.c
@@ -84,9 +84,7 @@ static const struct speed_map speeds[] = {
 #ifdef B1000000
 	{B1000000, 1000000/200 + 0x8000u},
 #endif
-#ifdef B1500000
 	{B1500000, 1500000/200 + 0x8000u},
-#endif
 #ifdef B2000000
 	{B2000000, 2000000/200 + 0x8000u},
 #endif
@@ -142,6 +140,9 @@ speed_t FAST_FUNC tty_value_to_baud(unsigned int value)
 {
 	int i = 0;
 
+	if (value == 1500000)
+		value = 1499904;
+
 	do {
 		if (value == tty_baud_to_value(speeds[i].speed)) {
 			return speeds[i].speed;
-- 
2.14.2

